src/test/KnownTestData.h

```diff
@@ -0,0 +1,276 @@
+                        "TakerPaysCurrency" : "0000000000000000000000004555520000000000",
+                        "TakerPaysIssuer" : "2ADB0B3959D60A6E6991F729E1918B7163925230"
+                    },
+                    "LedgerEntryType" : "DirectoryNode",
+                    "LedgerIndex" : "BC05A0B94DB6C7C0B2D9E04573F0463DC15DB8033ABA85624D0CE3CBADF66AFE"
+                }
+                }
+            ],
+            "TransactionIndex" : 21,
+            "TransactionResult" : "tesSUCCESS"
+        })",
+        "201C00000015F8E311006F567EAEE1B418DC7C00FC41E8DE6BA4FC0D79"
+        "CD6F7476D44B3D99B2346F3A78FE96E824004212E73400000000000000"
+        "FB5010BC05A0B94DB6C7C0B2D9E04573F0463DC15DB8033ABA85624D0C"
+        "E3BAC8E0468B64D54A4FC8A0B36BA20000000000000000000000004555"
+        "5200000000002ADB0B3959D60A6E6991F729E1918B7163925230654000"
+        "0012A05F20008114B100B28F60C3A425467387913CC0B297D3DA702CE1"
+        "E1E411006F568C2BEAAC384B373313F4E3E736A1C933B40E1AACB9D78B"
+        "9F363BBF6D9A4CCA60E7220000000024004212E12501B7A4F933000000"
+        "00000000003400000000000000FB5570F1E60F21B2A49ECBFE2D19E286"
+        "7E28D7C8F210E111A58F1C68541F71FBCFE35010BC05A0B94DB6C7C0B2"
+        "D9E04573F0463DC15DB8033ABA85624D0CE3CBADF66AFE64D54A4FD624"
+        "C5226500000000000000000000000045555200000000002ADB0B3959D6"
+        "0A6E6991F729E1918B71639252306540000012A05F20008114B100B28F"
+        "60C3A425467387913CC0B297D3DA702CE1E1E51100612501B7A4F9559C"
+        "D12991D25C792CA14DE4FAFCA1E55A59CA82BA7F1E045BCC212807CCBC"
+        "C9285694BAA26006FDE92FFDFD1EBBD162F38532C411004A04842C9DAC"
+        "0E37FF24F371E624004212E7624000001851148A53E1E7220000000024"
+        "004212E82D000000176240000018511489358114B100B28F60C3A42546"
+        "7387913CC0B297D3DA702CE1E1E511006456A283077E7F53AEBE1EBE2F"
+        "A6A5302C0594B34B6B6C04CB4972678B46F688756AE722000000003200"
+        "0000000000000058FF060B902D93027D3C91161AB9534D1144ABAA309C"
+        "2FD848783F5A3B9A11A80C8214B100B28F60C3A425467387913CC0B297"
+        "D3DA702CE1E1E511006456BC05A0B94DB6C7C0B2D9E04573F0463DC15D"
+        "B8033ABA85624D0CE3BAC8E0468BE72200000000364D0CE3BAC8E0468B"
+        "58BC05A0B94DB6C7C0B2D9E04573F0463DC15DB8033ABA85624D0CE3BA"
+        "C8E0468B0111000000000000000000000000455552000000000002112A"
+        "DB0B3959D60A6E6991F729E1918B716392523003110000000000000000"
+        "0000000000000000000000000411000000000000000000000000000000"
+        "0000000000E1E1E411006456BC05A0B94DB6C7C0B2D9E04573F0463DC1"
+        "5DB8033ABA85624D0CE3CBADF66AFEE72200000000364D0CE3CBADF66A"
+        "FE58BC05A0B94DB6C7C0B2D9E04573F0463DC15DB8033ABA85624D0CE3"
+        "CBADF66AFE011100000000000000000000000045555200000000000211"
+        "2ADB0B3959D60A6E6991F729E1918B7163925230031100000000000000"
+        "0000000000000000000000000004110000000000000000000000000000"
+        "000000000000E1E1F1031000"
+    };
+    return testMeta;
+}
+
+#if 0
```

Comment:
I was imagining we might support it someday, but it makes more sense to take it out and add it at that time.